modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -434,29 +420,29 @@ GrainTracker::trackGrains()
    */
   for (unsigned int map_num = 0; map_num < _maps_size; ++map_num)
     for (unsigned int i = 0; i < _feature_sets[map_num].size(); ++i)
-      if (_feature_sets[map_num][i]->_status == NOT_MARKED)
+      if (_feature_sets[map_num][i] && _feature_sets[map_num][i]->_status == NOT_MARKED)
```

Comment:
Ok, follow up to the question above. Do you need the null check here because you may have moved ownership of teh pointer away from `_feature_sets[map_num][i]`?